rllib/policy/torch_policy.py

```diff
@@ -202,9 +204,11 @@ def compute_actions_from_input_dict(
         with torch.no_grad():
             # Pass lazy (torch) tensor dict to Model as `input_dict`.
             input_dict = self._lazy_tensor_dict(input_dict)
+            # Pack internal state inputs into (separate) list.
             state_batches = [
-                input_dict[k] for k in input_dict.keys() if "state_" in k[:6]
+                input_dict[k] for k in input_dict.keys() if "state_in" in k[:8]
```

Comment:
safety-hook: Don't include accidentally present `state_out_x` keys (in the input_dict): This would actually not be caught anywhere and leads to the LSTM not learning.